ingestion/src/metadata/ingestion/source/pipeline/databrickspipeline/metadata.py

```diff
@@ -92,7 +92,7 @@ def get_pipeline_name(self, pipeline_details: dict) -> str:
         """
         Get Pipeline Name
         """
-        return pipeline_details["settings"]["name"]
+        return pipeline_details.get("settings").get("name")
```

Comment:
Chaining the `get` methods together doesn't actually help that much here. It's better to have a proper conditional logic in case something returns `None`